dbrequests/mysql/tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+"""Configures the test environment. Some helper functions and fixtures."""
```

Comment:
in the sake of DRY, lets think about pulling that from the initial conftest, since there is no change? you should be able to import the other test configuration

Reply:
Okay. I can try.

In the `dbrequests.tests.conftest` the Connection and Database class are imported. How can I parameterize to allow for the new mysql classes? If there were only functions, I would simply pass them as argument. 

I tried to apply what I found here: https://docs.pytest.org/en/latest/example/parametrize.html but I failed to get this to work. I guess the approach is to simply add an argument to the fixture. But I do not understand how I can pass an argument to such a fixtures.

Reply:
I dont know how to solve this and dont have the time right now to dig into it. I also would go for parametrizing the fixture, if this is not working, I would suggest letting it go for now.

Why are the tests so slow? Is it because of reset or because writing the tmpfile takes so long???

Reply:
The tests themselves are pretty fast, but the container with the test database takes a while for booting. There is a sleep of 50 seconds or so, and it is there for the tests of the base class and the mysql module.

> I dont know how to solve this and dont have the time right now to dig into it.

Okay. Then let's keep it the way it is right now. I will get back to it once I have a better understanding of the test infrastructure.

Reply:
Now I understand why these tests are slow. Now I also understand what (scope="module") means for a fixture.

Reply:
Ah, I overlooked the missing scope as well.